spectral_cube/tests/test_subcubes.py

```diff
@@ -150,3 +150,17 @@ def test_ds9region_new(regfile, result):
     #region = 'circle(2,2,2)'
     #subcube = cube.subcube_from_ds9region(region)
 
+
+@pytest.mark.skipif('not regionsOK', reason='Could not import regions')
+def test_regions():
+    cube, data = cube_and_raw('adv.fits')
+
+    # content of image.reg
+    regpix = regions.RectanglePixelRegion(regions.PixCoord(0.5, 1), width=4, height=2)
+
+    regpix.meta['range'] = [-318 * u.km/u.s, -320 * u.km/u.s]
```

Comment:
This looks fine, just make sure you confirm it's in range (I don't remember the velocity range of `adv.fits`...)

Reply:
I have checked the range and set the values accordingly.